task1.js

```diff
@@ -0,0 +1,14 @@
+import { Transform } from 'stream'
+
+const reverse = str => {
+    return str.split('').reverse().join('') + '\n\n\n'
+}
+
+const reverseTransform = new Transform({
+  transform(chunk, encoding, callback) {
+    this.push(reverse(chunk.toString().trim()));
+    callback();
+  }
+});
+
+process.stdin.pipe(reverseTransform).pipe(process.stdout)
```

Comment:
You could also use readline here for simplicity: https://nodejs.org/api/readline.html